crates/holochain_integrity_types/src/record.rs

```diff
@@ -271,14 +271,15 @@ impl<A> Record<A> {
     }
 }
 
-#[cfg(feature = "test_utils")]
+#[cfg(feature = "hashing")]
 impl<T> SignedHashed<T>
 where
     T: HashableContent,
     <T as holo_hash::HashableContent>::HashType: holo_hash::hash_type::HashTypeSync,
 {
-    /// Create a new signed and hashed content by hashing the content.
-    pub fn new(content: T, signature: Signature) -> Self {
+    /// Create a new signed and hashed content by hashing the content, but without checking
+    /// the signature.
+    pub fn new_unchecked(content: T, signature: Signature) -> Self {
```

Comment:
this is a mere rename to reflect what's already the case, yes?

Reply:
Yes, I want users to be sure they know that this is the case. This was one of the things that was originally intended to be test-only because it has the ability to construct an invalid value if used improperly, but I'm allowing it here because it's useful for efficiently rebuilding the struct from its parts, which does happen in one place in production. The name change means "beware".